mkdocs.yml

```diff
@@ -73,6 +73,9 @@ pages:
   # API Changes: api/changes.md                 # page-tree = NEED_NEW_PAGE
 - Hooks:
   - Using hooks: hooks.md                        # page-tree = NEED_PAGE_MOVE to /hooks/usage.md
```

Comment:
@mickadoo what do you think about this instead of the `- Enabling Hooks:` section? 

```yaml
  - Setup:
    - Hooks with Drupal: hooks/setup/drupal.md
    - Hooks with Joomla: hooks/setup/joomla.md
```

Reason: When I see "Enabling hooks" and then lots of "*$word* hooks", I think "Enabling hooks" will have pages for specific hooks which enable things.